samples/excel/89-preview-apis/chart-formatting.yaml

```diff
@@ -0,0 +1,328 @@
+id: excel-chart-formatting
+name: Chart - Formatting
+description: Format labels and lines of a slope chart.
+host: EXCEL
+api_set:
+    ExcelAPI: 1.8
+script:
+    content: |+
+        $("#addSampleData").click(() => tryCatch(addSampleData));
+        $("#create-chart").click(() => tryCatch(createChart));
+        $("#format-chart").click(() => tryCatch(formatChart));
+        $("#show-datalabels").click(() => tryCatch(dataLabels));
+        $("#change-color").click(() => tryCatch(changeColor));
+        $("#clear").click(() => tryCatch(clearChart));
+
+        async function createChart() {
+            await Excel.run(async (context) => {
+                var worksheet = context.workbook.worksheets.getActiveWorksheet();
+                var chart = worksheet.charts.add(Excel.ChartType.lineMarkers, worksheet.getRange("B6:G15"), Excel.ChartSeriesBy.rows);
+                chart.axes.categoryAxis.setCategoryNames(worksheet.getRange("C5:G5"));
+                chart.name = "SlopeChart";
+
+                // place chart below sample data
+                chart.top = 275;
+                chart.left = 50;
+                chart.height = 600;
+                chart.width = 850;
+                
+                chart.title.text = "GDP Per Capita , 1980 to 2017";
+                chart.legend.position = "Bottom";
+
+                worksheet.onChanged.add(onDataChanged);
+            });
+        }
+
+        async function formatChart() {
+            await Excel.run(async (context) => {
+                var worksheet = context.workbook.worksheets.getActiveWorksheet();
+                var chart = worksheet.charts.getItem("SlopeChart");
+
+                chart.axes.valueAxis.tickLabelPosition = "None";
+                chart.plotArea.width = 800;
+                chart.plotArea.left = 80;
+
+                chart.title.format.font.size = 14.4;
+                chart.title.text = "GDP PER CAPITA , 1980 TO 2017"
+                chart.title.format.font.name = "Times New Roman";
+                chart.title.format.font.color = "#D9D9D9";
+
+                chart.legend.format.font.name = "Times New Roman";
+                chart.legend.format.font.color = "#D9D9D9";
+
+                chart.axes.categoryAxis.format.font.name = "Times New Roman";
+                chart.axes.categoryAxis.format.font.color = "#D9D9D9";
+                chart.axes.valueAxis.majorGridlines.format.line.color = "#585858";
+                chart.format.fill.setSolidColor("#404040");
+            });
```

Comment:
The point about having one last `await context.sync()` applies here, too; and to all the Excel.runs in the file.